appserver/ejb/ejb-rest-remoting/client/src/main/java/fish/payara/ejb/rest/client/RemoteEJBContext.java

```diff
@@ -0,0 +1,387 @@
+        }
+        
+        if (environment.contains(FISH_PAYARA_KEY_STORE)) {
+            clientBuilder.keyStore(getInstance(environment.get(FISH_PAYARA_KEY_STORE), KeyStore.class), getPassword(environment.get("keyStorePassword")));
+        }
+        
+        if (environment.containsKey(FISH_PAYARA_READ_TIMEOUT)) {
+            clientBuilder.readTimeout(getLong(environment.get(FISH_PAYARA_READ_TIMEOUT)).longValue(), MICROSECONDS);
+        }
+        
+        if (environment.contains(FISH_PAYARA_SCHEDULED_EXECUTOR_SERVICE)) {
+            clientBuilder.scheduledExecutorService(getInstance(environment.get(FISH_PAYARA_SCHEDULED_EXECUTOR_SERVICE), ScheduledExecutorService.class));
+        }
+        
+        if (environment.contains(FISH_PAYARA_SSL_CONTEXT)) {
+            clientBuilder.sslContext(getInstance(environment.get(FISH_PAYARA_SSL_CONTEXT), SSLContext.class));
+        }
+        
+        if (environment.contains(FISH_PAYARA_TRUST_STORE)) {
+            clientBuilder.trustStore(getInstance(environment.get(FISH_PAYARA_TRUST_STORE), KeyStore.class));
+        }
+        
+        if (environment.contains(FISH_PAYARA_WITH_CONFIG)) {
+            clientBuilder.withConfig(getInstance(environment.get(FISH_PAYARA_WITH_CONFIG), Configuration.class));
+        }
+        
+        return clientBuilder;
+    }
+    
+    private NamingException newNamingException(String name, Exception cause) {
+        NamingException namingException = new NamingException("Could not lookup :" + name);
+        namingException.initCause(cause);
+        
+        return namingException;
+    }
+    
+    private <T> T getInstance(Object value, Class<T> clazz) throws ClassNotFoundException, InstantiationException, IllegalAccessException {
+        if (clazz.isInstance(value)) {
+            return clazz.cast(value);
+        }
+        
+        if (value instanceof String) {
+            return clazz.cast(Class.forName((String)value).newInstance());
+        }
+        
+        throw new IllegalStateException("Value " + value + " has to be of type String or " + clazz);
+    }
+    
+    private Long getLong(Object value) {
+        if (value instanceof Number) {
+            return ((Number) value).longValue();
+        }
+        
+        if (value instanceof String) {
+            return Long.valueOf((String) value);
+        }
+        
+        throw new IllegalStateException("Value " + value + " has to be of type String or Number");
+    }
+    
+    private char[] getPassword(Object value) {
+        if (value instanceof String) {
+            return ((String) value).toCharArray();
+        }
+        
+        if (value instanceof char[]) {
+            return (char[]) value;
+        } 
+            throw new IllegalArgumentException("No password provided");
+        
+    }
+
+    @Override
+    public void bind(Name name, Object obj) throws NamingException {
+        
```

Comment:
Same as above, what are these empty methods doing?

Reply:
Nothing, but required to be there for the interface.